projects/cashmere/src/lib/input/hc-form-field.component.spec.ts

```diff
@@ -0,0 +1,165 @@
+/* tslint:disable:no-use-before-declare component-class-suffix */
```

Comment:
I ssee this tslint exception enough I think it could be disabled from the .tslint file

Reply:
Which are you referring to? For tests it's alright but I'd like to have that for the rest of the app